wiki/en/en-Installation-for-Linux.md

```diff
@@ -91,14 +91,14 @@ You can now delete the sources directory you compiled from.
 ### Configure JACK with QjackCtl
 Jamulus clients need [JACK](https://jackaudio.org/) to run, but you need to configure that first. The recommended method is to use `QjackCtl`.
 1. Open the command shell e.g. with Ctrl-Alt-T and
-1. Execute the command `qjackctl` you will see the **JACK Audio Connection Kit**
+1. Execute the command `qjackctl`. You will see the **JACK Audio Connection Kit**
```

Comment:
Hm.  No, you won't.  You'll see the Qt JACK Control utility main page.

Reply:
Done